auth/src/main/java/software/amazon/awssdk/auth/signer/params/Aws4PresignerParams.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.auth.signer.params;
+
+import java.util.Date;
+
+public final class Aws4PresignerParams extends Aws4SignerParams {
```

Comment:
Should this be internal?

Reply:
Nope, Used in overloaded sign methods in both signer implementations